packages/paste-core/components/table/src/Table.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {TableProps, TablePropTypes} from './types';
+
+const Table = React.forwardRef<HTMLTableElement, TableProps>(({variant = 'auto', ...props}, ref) => {
```

Comment:
Just wondering if, for simplicity, we should call the `variant` prop `tableLayout`.  Similarly for the other parts of the table: `alignment` -> `textAlign`, `verticalAlignment` -> `verticalAlign`.

Reason being that it's a familiar API for folks who know how HTML tables work and we aren't really doing anything special with these props.  What do you think?

Reply:
I like that better. More familiar API, and it also opens up `variant` if we ever need that in the future for say an inverse table.